examples/fluids/navierstokes.h

```diff
@@ -97,6 +97,11 @@ extern PetscLogEvent FLUIDS_CeedOperatorApply;
 extern PetscLogEvent FLUIDS_CeedOperatorAssemble;
 extern PetscLogEvent FLUIDS_CeedOperatorAssembleDiagonal;
 extern PetscLogEvent FLUIDS_CeedOperatorAssemblePointBlockDiagonal;
+extern PetscLogEvent FilterData;
```

Comment:
Doesn't look like you have a implemented logging for `FilterData`. I'd either add it in or remove this line; I'm fine with either.

Reply:
Timing for the filtering should probably be added for other type of performance considerations. Up to you if you would like to see this data. I will remove it for this PR to keep it SmartRedis specific.

Reply:
Yeah, I can add it in after this PR gets merged.